utils/format.go

```diff
@@ -334,25 +334,26 @@ func FormatBlockStatusShort(status uint64) template.HTML {
 
 func FormatBlockStatusStyle(status uint64) template.HTML {
 	if status == 0 {
-		return `<div title="This slot is still scheduled" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-neutral-1"><p class="style-status-tag-text" >S<span class="d-none d-sm-inline">cheduled</span></p></div>`
+		return `<div title="This slot is still scheduled" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-neutral-1"><div class="style-status-tag-text" >S<span class="d-none d-sm-inline">cheduled</span></div></div>`
 	} else if status == 1 {
-		return `<div title="This block has been proposed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-good"><p class="style-status-tag-text text-white" >P<span class="d-none d-sm-inline">roposed</span></p></div>`
+		return `<div title="This block has been proposed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-good"><div class="style-status-tag-text text-white" >P<span class="d-none d-sm-inline">roposed</span></div></div>`
 	} else if status == 2 {
-		return `<div title="This block has been proposed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-bad"><p class="style-status-tag-text" >M<span class="d-none d-sm-inline">issed</span></p></div>`
+		return `<div title="This blocks proposal has been missed" data-toggle="tooltip" class="style-badge style-badge-single-char style-bg-bad"><div class="style-status-tag-text" >M<span class="d-none d-sm-inline">issed</span></div></div>`
```

Comment:
Missing the apostrophe, should be "block's". Or change it to "block proposal" which would make sense too imho.